### DISCUSSION TITLE
Low and High res RTSP stream

### DISCUSSION THREAD

**Discussion by mspinolo** [roleoroleo/yi-hack-Allwinner-v2#126] [Q&A]
Hi there,

I am on latest 0.15 and can’t find how to enable both rtsp streams (high and low res): I read the documentation but couldn’t find any hint

**Comment by roleoroleo** [comment]
In Configuration page set "Both".

![immagine](https://user-images.githubusercontent.com/39277388/114170148-c8ba5000-9932-11eb-9d68-8965dc5b8cb9.png)
